cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc

```diff
@@ -6,9 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use infrastructure nodes to isolate infrastructure workloads for two primary purposes:
+Infrastructure nodes prevent additional billing cost against subscription counts. 
```

Comment:
I recommend preserving the user-centered tone and explanation in the original phrase even though the detail have changed.

Reply:
```suggestion
Infrastructure nodes prevent additional billing costs against subscription counts. 
```